### ISSUE TITLE
The docker image is very large 14GB on disk.

### ISSUE THREAD

**Issue by ianpurton** [go-skynet/LocalAI#1162]
The image is so large that I can't run a Github action as it runs out of space. 

```sh
quay.io/go-skynet/local-ai v1.30.0  ba9a2b1bff11   2 weeks ago   14GB
```

Using dive I can see the image has Cuda installed and this takes up a lot of space.

## Possible solution

Have a CPU only image with no Cuda?

**Comment by mudler** [comment]
fair point, the images got larger as backends were added, and I guess it will just get fatter due to #1144 .

We could - at least - split the images to a core and extra version, which comes with all the python backends as well.

**Comment by mudler** [comment]
this is still much wanted as today - we have now a flag to build smaller images (`IMAGE_TYPE=core`) but needs to be added to the pipelines

**Comment by mudler** [comment]
https://github.com/mudler/LocalAI/pull/1309 introduces `-core` images, let's check what builds out of master and see if we can improve the size of those
